src/test/suite/current-file-path.ts

```diff
@@ -0,0 +1,9 @@
+import * as vscode from 'vscode';
```

Comment:
By doing this you import all the exported element of vscode, consider using: `import { window } from 'vscode'` instead